internal/npm_install/npm_install.bzl

```diff
@@ -83,6 +83,15 @@ fine grained npm dependencies.
         default = True,
         doc = "If stdout and stderr should be printed to the terminal.",
     ),
+    "strict_visibility": attr.bool(
+        default = False,
+        doc = """Turn on stricter visibility for generated BUILD.bazel files
```

Comment:
mention that the default will likely change to True in 3.0 (and also file an issue on the 3.0 milestone to remind us to flip it)